newrelic/src/main/java/com/cerner/beadledom/newrelic/NewRelicApi.java

```diff
@@ -0,0 +1,19 @@
+package com.cerner.beadledom.newrelic;
+
+/**
+ * New Relic API. The API mimics the actual New Relic API, however without the static methods.
+ *
+ * @author Nathan Schile
+ * @since 2.8
+ */
+public interface NewRelicApi {
+
+  /**
+   * Add a key/value pair to the current transaction.
+   *
+   * @param key Custom parameter key.
```

Comment:
No period at the end of params.

Reply:
Removed period, https://github.com/cerner/beadledom/pull/93/commits/26450f729643a5f80e29be58e15817f4a67bf2eb